Android/Nothing/app/src/main/java/com/gorro/nothing/MonthAndDay.kt

```diff
@@ -10,3 +13,10 @@ typealias MonthDay = Pair<Int, Int>
  * @return true if December 24th or December 25th
  */
 fun MonthDay.isItChristmas(): Boolean = first == 11 && (second == 24 || second == 25)
+
+
+fun getCurrentMonthAndDay(): MonthDay {
+    val calendar = Calendar.getInstance()
+    Log.e("Calendar", calendar.toString())
```

Comment:
remove debug code before sending PR

Reply:
This was just migrations from @gorrotowi if the remove is needed is under his watch

Reply:
This is not a debug code, it's a feature. ✌️